portal-impl/src/com/liferay/portal/security/permission/ResourceActionsImpl.java

```diff
@@ -825,6 +825,8 @@ private ResourceActionsBag _getPortletResourceActionsBag(String name) {
 
 			_checkPortletActions(portlet, portletActions);
 
```

Comment:
@dantewang I did not include this cause
1. the logic is kind of duplicated in _checkPortletActions(). 
2. Just look at the code, should you always add this permission to portlet? My answer is NO. But currently the real logic is like this because history bugs. Even if you did not change any logic here, it does not mean this is right. Please look at my changes, I removed the checking in if condition, it makes more sense to me.

Tina.